tests/core/unit/vdom.ts

```diff
@@ -3430,6 +3430,42 @@ jsdomDescribe('vdom', () => {
 				assert.isTrue(consoleWarnStub.calledOnce);
 			});
 
+			it('typed children', () => {
```

Comment:
do we need an example text with `w()` also?

Reply:
I can add that

Reply:
Added.